template/index.js

```diff
@@ -0,0 +1,58 @@
+import { File } from "@asyncapi/generator-react-sdk";
+
+// Import custom components from file
+import { HTML, Head, Body } from "../components/common";
+
+export default function Index({ asyncapi, params }) {
+  const getCssLinks = () => {
+    const cssLinks = [];
+    if (params.singleFile) {
+      cssLinks.push("template/css/asyncapi-ui.css");
+      cssLinks.push("template/css/asyncapi-ui.css");
+    } else {
+      cssLinks.push("css/asyncapi-ui.css");
+      cssLinks.push("css/asyncapi-ui.css");
+    }
+    return cssLinks;
+  };
```

Comment:
That function is unnecessary, you can perform that logic without function. Also you have here big bug. In the original template https://github.com/asyncapi/html-template/blob/master/template/index.html#L12 you can see that if `singleFile` param is set you should pass here the content of the given file (using that function https://github.com/asyncapi/html-template/blob/master/filters/all.js#L55) not css link, and also we have two kinds of css files `global.min.css` and `asyncapi.min.css`. Please fix that.